Run multiple instances on same server

<!-- Please describe your issue in English. -->

#### Can be reproduced ?
Yes

#### Steps:
1. Run a mo-server instance
2. then, start another new mo-server instance

#### Expected behavior:
Threw an error instead of new instance crash

#### Actual behavior:
```
[devops@mo-test2 scratch]$ ./mo-server system_vars_config.toml
2021/09/24 16:16:59.489835 [info] prophet: prophet logger set
Shutdown The Server With Ctrl+C | Ctrl+\.
2021/09/24 16:16:59.493445 +0800 ERROR logutil/api.go:63 error locking file: resource temporarily unavailable
matrixone/pkg/logutil.Errorf
	/home/devops/stage/matrixone/pkg/logutil/api.go:63
matrixone/pkg/vm/engine/aoe/storage/db.createDBLock
	/home/devops/stage/matrixone/pkg/vm/engine/aoe/storage/db/lock.go:50
matrixone/pkg/vm/engine/aoe/storage/db.Open
	/home/devops/stage/matrixone/pkg/vm/engine/aoe/storage/db/open.go:33
matrixone/pkg/vm/driver/aoe.NewStorageWithOptions
	/home/devops/stage/matrixone/pkg/vm/driver/aoe/storage.go:54
main.main
	/home/devops/stage/matrixone/cmd/db-server/main.go:165
runtime.main
	/usr/local/go/src/runtime/proc.go:255
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1c97290]

goroutine 1 [running]:
matrixone/pkg/vm/driver.NewCubeDriverWithFactory({0x26be440, 0x0}, {0x26a73b0, 0x0}, {0x26a93e0, 0x0}, 0xc0001d7800, 0x2469518)
	/home/devops/stage/matrixone/pkg/vm/driver/cube_driver.go:145 +0xd0
matrixone/pkg/vm/driver.NewCubeDriverWithOptions({0x26be440, 0x0}, {0x26a73b0, 0x0}, {0x26a93e0, 0x0}, 0x35da6e0)
	/home/devops/stage/matrixone/pkg/vm/driver/cube_driver.go:130 +0x3e
main.main()
	/home/devops/stage/matrixone/cmd/db-server/main.go:187 +0x979
```
#### Environment:
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:

#### Configuration file:


#### Additional context:
- Error message from client:
- Server log:
- Other information:

--- a/cmd/db-server/main.go
+++ b/cmd/db-server/main.go
@@ -133,22 +133,23 @@ func main() {
 	log.SetHighlighting(false)
 	util.SetLogger(log.NewLoggerWithPrefix("prophet"))
 
+	logutil.SetupMOLogger(os.Args[1])
+
 	//before anything using the configuration
 	if err := config.GlobalSystemVariables.LoadInitialValues(); err != nil {
-		fmt.Printf("Initial values error:%v\n", err)
+		logutil.Infof("Initial values error:%v\n", err)
 		os.Exit(InitialValuesExit)
 	}
 
 	if err := config.LoadvarsConfigFromFile(os.Args[1], &config.GlobalSystemVariables); err != nil {
-		fmt.Printf("Load config error:%v\n", err)
+		logutil.Infof("Load config error:%v\n", err)
 		os.Exit(LoadConfigExit)
 	}
 
-	fmt.Println("Shutdown The Server With Ctrl+C | Ctrl+\\.")
+	logutil.Infof("Shutdown The Server With Ctrl+C | Ctrl+\\.")
 
 	config.HostMmu = host.New(config.GlobalSystemVariables.GetHostMmuLimitation())
 
-	logutil.SetupMOLogger(os.Args[1])
 	log.SetLevelByString(config.GlobalSystemVariables.GetCubeLogLevel())
 
 	Host := config.GlobalSystemVariables.GetHost()
@@ -162,7 +163,7 @@ func main() {
 
 	targetDir := config.GlobalSystemVariables.GetCubeDirPrefix() + strNodeId
 	if err := recreateDir(targetDir); err != nil {
-		fmt.Printf("Recreate dir error:%v\n", err)
+		logutil.Infof("Recreate dir error:%v\n", err)
 		os.Exit(RecreateDirExit)
 	}
 
@@ -177,26 +178,26 @@ func main() {
 	opt := aoeStorage.Options{}
 	_, err = toml.DecodeFile(os.Args[1], &opt)
 	if err != nil {
-		fmt.Printf("Decode aoe config error:%v\n", err)
+		logutil.Infof("Decode aoe config error:%v\n", err)
 		os.Exit(DecodeAoeConfigExit)
 	}
 
 	aoeDataStorage, err = aoeDriver.NewStorageWithOptions(targetDir+"/aoe", &opt)
 	if err != nil {
-		fmt.Printf("Create aoe driver error, %v\n", err)
+		logutil.Infof("Create aoe driver error, %v\n", err)
 		os.Exit(CreateAoeExit)
 	}
 
 	cfg := dConfig.Config{}
 	_, err = toml.DecodeFile(os.Args[1], &cfg.CubeConfig)
 	if err != nil {
-		fmt.Printf("Decode cube config error:%v\n", err)
+		logutil.Infof("Decode cube config error:%v\n", err)
 		os.Exit(DecodeCubeConfigExit)
 	}
 
 	_, err = toml.DecodeFile(os.Args[1], &cfg.ClusterConfig)
 	if err != nil {
-		fmt.Printf("Decode cluster config error:%v\n", err)
+		logutil.Infof("Decode cluster config error:%v\n", err)
 		os.Exit(DecodeClusterConfigExit)
 	}
 	cfg.ServerConfig = server.Cfg{
@@ -211,12 +212,12 @@ func main() {
 
 	a, err := driver.NewCubeDriverWithOptions(metaStorage, pebbleDataStorage, aoeDataStorage, &cfg)
 	if err != nil {
-		fmt.Printf("Create cube driver failed, %v", err)
+		logutil.Infof("Create cube driver failed, %v", err)
 		os.Exit(CreateCubeExit)
 	}
 	err = a.Start()
 	if err != nil {
-		fmt.Printf("Start cube driver failed, %v", err)
+		logutil.Infof("Start cube driver failed, %v", err)
 		os.Exit(StartCubeExit)
 	}
 	c = catalog.NewCatalog(a)
@@ -238,7 +239,7 @@ func main() {
 		log.SetLevel(logger.WARN)*/
 	srv, err := rpcserver.New(fmt.Sprintf("%s:%d", Host, 20100+NodeId), 1<<30, logutil.GetGlobalLogger())
 	if err != nil {
-		fmt.Printf("Create rpcserver failed, %v", err)
+		logutil.Infof("Create rpcserver failed, %v", err)
 		os.Exit(CreateRPCExit)
 	}
 	hp := handler.New(eng, proc)
@@ -247,7 +248,7 @@ func main() {
 	err = waitClusterStartup(a, 300*time.Second, int(cfg.CubeConfig.Prophet.Replication.MaxReplicas), int(cfg.ClusterConfig.PreAllocatedGroupNum))
 
 	if err != nil {
-		fmt.Printf("wait cube cluster startup failed, %v", err)
+		logutil.Infof("wait cube cluster startup failed, %v", err)
 		os.Exit(WaitCubeStartExit)
 	}
 
@@ -262,7 +263,7 @@ func main() {
 
 	err = runMOServer()
 	if err != nil {
-		fmt.Printf("Start MOServer failed, %v", err)
+		logutil.Infof("Start MOServer failed, %v", err)
 		os.Exit(StartMOExit)
 	}
 	//registerSignalHandlers()

